### DISCUSSION TITLE
Borked my Crowdsec install, how to uninstall and reinitialize?

### DISCUSSION THREAD

**Discussion by BenAgin1** [tteck/Proxmox#2139] [Help Needed] [answered]
So I was trying to add edit the config.yaml file to get rid of the use_wal error, but I somehow did something to prevent crowdstrike from starting up again. I thought I could delete /crowdsec directory and rerun the tteck script, but it does not come back...

**Comment by tteck** [comment, marked as answer]
Have you tried the command `apt-get autoremove --purge crowdsec`?
